docs/tutorial/plot_bbegin_measure_time.py

```diff
@@ -122,7 +122,7 @@
 
 
 df = DataFrame(obs)
-df
+print(df)
```

Comment:
I let it that that because the rendering is better in the documentation.

Reply:
got it. I reversed this change.

Reply:
got it. I reversed this change.